### ISSUE TITLE
With ScrollY enabled, ScrollX cannot be disabled

### ISSUE THREAD

**Issue by statquant** [rstudio/DT#697]
Hello when using the "Scroller" extension, it seems there is no way to disable `ScrollX` while there is a working `ScrollY` argument.
That would be useful on very wide tables, would that be possible to change this ?
Many thanks for this great package

**Comment by yihui** [comment]
Please follow the issue guide and mention cross-posting: https://stackoverflow.com/q/57468216/559676

**Comment by statquant** [comment]
Right sorry, I posted late on SO (6 hours latter) and did not remembered to indicate it here.
SO suggestion is not really a fix per say, do we agree it is a bug or do I close this issue ?

**Comment by yihui** [comment]
No worries. Sorry I don't have time to investigate this issue. Let's see if others can help. Thanks!

**Comment by shrektan** [comment]
@statquant Would you mind provide a minimal example? And please verify behavior exists outside of RStudio, which seems will automatically add the scrollX somehow.

**Comment by shrektan** [comment]
I believe there's nothing to do with Scroller extention. It seems ScrollY will force to enable ScrollX.

```r
library(DT)
DF = data.frame(x = 1:100, y = rep("a really really really really really really really really really really really really really really really really really really really really really really really really really really really really really really really really really long line", 100))
datatable(
  DF, 
  options = list(
    scrollY = 100,
    scrollX = FALSE
  ),
  class = 'nowrap'
)
```

**Comment by shrektan** [comment]
I think it's an issue on [datatables library](http://datatables.net/). Beside the issues mentioned on stackoverflow (see [this](https://stackoverflow.com/questions/40134622/datatables-scrollx-wont-go-away-when-scrolly-enabled) and [this](https://stackoverflow.com/questions/32847344/vertical-scroll-settings-effecting-horizontal-scrolling-with-datatables)),  [this official example on ScrollY](https://datatables.net/examples/basic_init/scroll_y.html) will display ScrollX if you set your browser's width to very narrow.

In short, I don't see there's anything we can do now. The only workaround I can think of is:

1. as [your self-answered question](https://stackoverflow.com/q/57468216/559676), set a fixed large width;
2. or, set width to 100% (still displays if the width exceeds the browser but that should be desired in most cases...)

**Comment by shrektan** [comment]
@statquant I've created a [fiddle example](https://jsfiddle.net/obcqrz5f/). It's indeed an issue with datatables library itself.

**Comment by ramunozp** [comment]
Today the problem continues. Is there any solution?
